cernopendata/modules/fixtures/data/docs/cms-guide-luminosity-calculation/cms-guide-luminosity-calculation.md

```diff
@@ -0,0 +1,275 @@
+
+  ```
+  Successfully built brilws
+  Installing collected packages: brilws
+  Successfully installed brilws
+  ```
+
+3. Check the installation by running the command:
+
+  `brilcalc --version`
+
+   which should output `3.6.2`
+
+Now you are ready to [test `brilcalc`.](#test-brilcalc)
+
+### conda
+
+Alternatively you can install a conda environment:
+
+1. In the command prompt go to the home directory in our container with the command `cd`
+
+2. Fetch the installer for the `brilcalc` conda environment:
+
+ `wget https://cern.ch/cmslumisw/installers/linux-64/Brilconda-1.1.7-Linux-x86_64.sh`
+
+ (you may need to use the `--no-check-certificate` option with `wget`)
+
+3. Run the installer:
+
+ `bash Brilconda-1.1.7-Linux-x86_64.sh -b -p <localbrilcondabase>`
+
+ where we substitute `<localbrilcondabase>` with `brilconda`. This is the directory in which the `brilcalc` tools
+ will be installed.
+
+4. Once installed successfully add the tools to your `PATH` with the following command:
+
+ `export PATH=$HOME/.local/bin:$HOME/brilconda/bin:$PATH`
+
+5. Then install `brilcalc` with this command:
+
+ `pip install brilws`
+
+6. Finally check by running the following:
+
+ `brilcalc --version`
+
+  which should output `3.53`
+
+Now you are ready to [test `brilcalc`.](#test-brilcalc)
+
+## Test brilcalc
+
+As a test, let's see what the integrated luminosity was for run 160431 by running the command
+
+ `brilcalc lumi -c web -r 160431`
+
+which should output
+
+```
+#Data tag : 19v3 , Norm tag: onlineresult
++-------------+-------------------+-----+------+----------------+----------------+
+| run:fill    | time              | nls | ncms | delivered(/ub) | recorded(/ub)  |
++-------------+-------------------+-----+------+----------------+----------------+
+| 160431:1615 | 03/14/11 03:08:07 | 243 | 218  | 8781.334977241 | 7879.289611261 |
++-------------+-------------------+-----+------+----------------+----------------+
+#Summary:
++-------+------+-----+------+-------------------+------------------+
+| nfill | nrun | nls | ncms | totdelivered(/ub) | totrecorded(/ub) |
++-------+------+-----+------+-------------------+------------------+
+| 1     | 1    | 243 | 218  | 8781.334977241    | 7879.289611261   |
++-------+------+-----+------+-------------------+------------------+
+```
+
+**Note:** It is important that you use the `-c web` option when running `brilcalc`. This specifies that you use indirect access to BRIL servers via web cache. For users of CMS open data outside CERN and CMS this is the only option that will work.
+
+**Note:** There is a useful `help` option for `brilcalc` and its commands:
+
+`brilcalc --help`
+
+```
+usage:
+   brilcalc (-h|--help|--version)
+   brilcalc [--debug|--warn] <command> [<args>...]
+
+ commands:
+   lumi Luminosity
+   beam Beam
+   trg Trigger configurations
+ See 'brilcalc <command> --help' for more information on a specific command.
+ ```
+
+## Common queries
```

Comment:
I am not sure why, but this is not rendering well neither.  I can see the hashtags (##) and not as headings from here below.  Maybe the ``` or the `` are battling each other. 
![temp](https://user-images.githubusercontent.com/8090959/91122359-c1725a80-e65f-11ea-9588-a53f00669c53.png)

Reply:
Same as comment above. Fine for Github and atom